test/unit/test_features.py

```diff
@@ -217,7 +243,7 @@ def test_explicit_detect_features(self):
 
   def test_emit_all_features(self):
     p = run_process(WASM_OPT + ['--emit-target-features', '-all', '-o', '-'],
-                    input="(module)", check=False, capture_output=True)
+                    input="(module)", check=False, capture_output=True, decode_output=False)
```

Comment:
This is for a problem that stumped me for a while. What _seems_ to have happened is that previously the code below would always try to decode as utf-8 however with the new addition of one more feature flag, it pushes the binary representation of the user section **size** over 127 (it was literally 127 exactly before) causing it to be "malformed" utf-8. Correct me if I'm wrong.

I'm not familiar enough with all the moving pieces of the testing suite to know if there's a more elegant solution. I'm all ears!

Reply:
Yes, I think this method was written assuming the output is text. While I'm not sure if adding the option is the best way to fix this, we actually have another similar method [`run_command`](https://github.com/WebAssembly/binaryen/blob/69ad1e8a8d2e1d395e30230433742f4f5668563b/scripts/test/support.py#L165-L184) elsewhere, so I guess future refactoring should combine these methods or something and also take care of both text and binary output. I don't think that should be included in this PR, of course.

So, I'm not very sure about what the best quick fix should be :) I don't mind much about the current option, but maybe cc @tlively

Reply:
I would prefer a solution that just never decodes the output instead of adding another parameter. I’m fine with doing that cleanup in a separate PR, though, if it touches a bunch of unrelated tests.